### PR TITLE
missing make_corpus_subset.sh 

### PR DESCRIPTION
**Pull request by johnjosephmorgan**
The run-4-anydecod-langs.sh script calls local/make_corpus_subset.sh, but this script is not there. I copied the script from babel/s5d.

### CODE REVIEW

**Comment by danpovey** [comment]
@jtrmal, can you look at this?

**Comment by jtrmal** [comment]
Ok. Will do.
Y.

File: egs/wsj/s5/local/chain/tuning/run_tdnn_1b.sh [hunk 1]
```diff
@@ -131,7 +131,7 @@ fi
 if [ $stage -le 13 ]; then
   # Get the alignments as lattices (gives the chain training more freedom).
   # use the same num-jobs as the alignments
-  steps/align_fmllr_lats.sh --nj 100 --cmd "$train_cmd" ${lores_train_data_dir} \
+  steps/align_fmllr_lats.sh --nj $nj --cmd "$train_cmd" ${lores_train_data_dir} \
```
Comment (jtrmal) [inline egs/wsj/s5/local/chain/tuning/run_tdnn_1b.sh]: @danpovey @johnjosephmorgan  I know there was a conversation on this and I think at the end the sentiment was to leave 100 here?

**Comment by jtrmal** [comment]
I  think it's pretty harmless when you do squash-n-merge (after addressing the $nj vs 100). It has a lot of revisions because John didn't based it on the current head.

**Comment by danpovey** [comment]
Yes, leave 100.

**Comment by danpovey** [comment]
@johnjosephmorgan, can you change the --nj $nj back to --nj 100?

**Pull request merged**